File: Friendly Array/Friendly_Array.cpp
class Solution {
  public:
    int calculateFriendliness(vector<int>& arr) {
        int ans =0;
        int n = arr.size();
        for(int i =1;i<n; i++) {
            ans += abs(arr[i]-arr[i-1]);
        }
        ans += abs(arr[0]-arr[n-1]);
        return ans;
    }
};

File: Friendly Array/Friendly_Array.py
class Solution:
    def calculateFriendliness(self, arr):
        ans = 0
        n = len(arr)
        for i in range(1, n):
            ans += abs(arr[i] - arr[i - 1])
        ans += abs(arr[0] - arr[n - 1])
        return ans
